src/framework/core/js/Fluid.js

```diff
@@ -2321,26 +2364,27 @@ var fluid = fluid || fluid_3_0_0;
         return !source ? undefined : source[seg];
     };
 
-    /** Merge a collection of options structures onto a target, following an optional policy.
+    /**
+     * Merge a collection of options structures onto a target, following an optional policy.
      * This method is now used only for the purpose of merging "dead" option documents in order to
      * cache graded component defaults. Component option merging is now performed by the
      * fluid.makeMergeOptions pathway which sets up a deferred merging process. This function
      * will not be removed in the Fluid 2.0 release but it is recommended that users not call it
      * directly.
      * The behaviour of this function is explained more fully on
      * the page http://wiki.fluidproject.org/display/fluid/Options+Merging+for+Fluid+Components .
+     *
      * @param {Object|String} policy - A "policy object" specifiying the type of merge to be performed.
      * If policy is of type {String} it should take on the value "replace" representing
      * a static policy. If it is an
      * Object, it should contain a mapping of EL paths onto these String values, representing a
      * fine-grained policy. If it is an Object, the values may also themselves be EL paths
      * representing that a default value is to be taken from that path.
-     * @param {...Object} options1, options2, .... - an arbitrary list of options structure which are to
+     * @param {...Object} sources - an arbitrary list of options structure which are to
      * be merged together. These will not be modified.
      */
 
-    fluid.merge = function (policy /*, ... sources */) {
-        var sources = Array.prototype.slice.call(arguments, 1);
+    fluid.merge = function (policy , ...sources) {
```

Comment:
@amb26 this is another case where I've converted to use a [rest parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters). In this case the other benefit is that the first parameter called `policy` is not included in the `sources` array, so we don't need to handle skipping it later in the code.